### PR TITLE
feat(logic): move data representation from tornado-gui to here

### PR DESCRIPTION
**Pull request by anarchuser**

### CODE REVIEW

**Comment by SKoschnicke** [comment]
👍🏻

**Comment by anarchuser** [comment]
Should I merge it like this already, or rather add more feats and tests first?

**Comment by SKoschnicke** [comment]
Would be nice if you add a skeleton of the missing classes.

File: plugin/src/server/sc/plugin2021/GamePlugin.kt [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package sc.plugin2021
+
+import sc.api.plugins.IGameInstance
+import sc.api.plugins.IGamePlugin
+import sc.api.plugins.host.IGamePluginHost
+import sc.plugin2020.GamePlugin
+import sc.plugins.PluginDescriptor
+import sc.shared.ScoreDefinition
+
+@PluginDescriptor(name = "Hive", uuid = GamePlugin.PLUGIN_UUID, author = GamePlugin.PLUGIN_AUTHOR)
```
Comment (SKoschnicke) [inline plugin/src/server/sc/plugin2021/GamePlugin.kt:10]: Careful with the game id!

Comment (anarchuser) [reply plugin/src/server/sc/plugin2021/GamePlugin.kt:10]: Ah yeah, I forgot...
On that note, I probably forgot to change some `plugin2020` to `plugin2021` as well.
Thanks for the note!

**Pull request merged**